### DISCUSSION TITLE
Cómo obtener la clave API de Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#2] [Announcements]
# Cómo obtener la clave API de Blogger

Aprende a obtener una clave de API de Blogger, muy útil para aplicaciones basadas en  la API 3.0 de Blogger

https://www.zkreations.com/2018/06/clave-de-api-para-blogger.html

**Comment by arcanjoip** [comment]
O blog ta incrível

**Comment by gianles** [comment]
Hola Daniel, buen articulo. Para que sirve crear una api para un blog de blogger?

**Reply by danieIabel** [reply]
Buenos dias amigo. Principalmente para trabajar con la api version 3 de Blogger, la cual es mucho mas rápida y optimizada, con ella puedes crear por ejemplo plugins, tambien te puede ser util para crear aplicaciones con Blogger, en este caso ya seria mas avanzado pero puedes ver un ejemplo de implementación en la documentación oficial de Blogger, en donde se proporciona una apikey de prueba: [developers.google.com/blogger/docs/3.0/](https://developers.google.com/blogger/docs/3.0/reference/posts/get?hl=es-419)

Como podrás notar es algo avanzado, pero en aportes que tengo pensado compartir solo necesitarás crear la api-key y ubicarla en donde indico, por eso cree este articulo para que sea facil de obtener.
